pegasus/sites.v3/code.org/public/volunteer/index.haml

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Volunteer for the Hour of Code"
+title: "Volunteer"
```

Comment:
not super clear, especially for og tags. Maybe 'Volunteer in Computer Science Education'?